src/connection/sim.ts

```diff
@@ -475,6 +494,21 @@ export class SimulatorPlugin implements Connection {
       initial = undefined;
       updateRate = 100;
       cls = RampPv;
+    } else if (nameInfo.protocol.includes("sim://flipflop")) {
+      // Default rate if not configured with flipflop(X)
+      updateRate = 1000;
+      const strCmd = nameInfo.protocol;
+      if (strCmd.includes("(") && strCmd.includes(")")) {
+        const intevalSecStr = strCmd.slice(
+          strCmd.indexOf("(") + 1,
+          strCmd.lastIndexOf(")")
+        );
+        if (intevalSecStr !== "") {
+          updateRate = parseFloat(intevalSecStr) * 1000;
```

Comment:
Typo in variable name inteval -> interval

Reply:
Fixed in latest commit.